src/main/java/org/zalando/pazuzu/feature/FeatureRepositoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.pazuzu.feature;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+public class FeatureRepositoryImpl implements FeatureRepositoryCustom {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Override
+    public List<Feature> getFeatures(int offset, int limit) {
```

Comment:
Ok, got it.

Have you thought about implementing pagination via Spring Data defaults: http://www.petrikainulainen.net/programming/spring-framework/spring-data-jpa-tutorial-part-seven-pagination/